README.md

```diff
@@ -624,6 +622,19 @@ WriteRequest{
 
 ### Privacy Considerations
 
+The `did:earth` method is a fully compliant IID and DID. As such the privacy considerations taken into account are applicable to this method. The reader is directed towards the privacy considerations discussed in [section 10 of the W3C DID Core specification](https://w3c.github.io/did-core/#privacy-considerations).
```

Comment:
Unfortunately, this paragraph doesn't say much.

What we should highlight are the following points
1. IIDs, e.g., `did:earth:ixo:nft:1:abc` are designed to represent on-chain assets, as such the identifier itself is never used to refer to real-world objects like people, with associated privacy requirements.
2. IID references, e.g., `did:earth:ixo:nft:1:abc#creator` may in fact be used to refer to real people and organizations. As such, care must be taken to ensure that any associated personal data be managed off chain with appropriate privacy mechanisms such as the ability to remove the data from public disclosure.
4. IID references specified as Linked Resources offer several privacy-agile ways to associate the on-chain asset with arbitrary resources, providing a way for `did:earth:ixo:nft:1:abc#creator` to refer to an off-chain data store with the sensitive data. For example, an NFT's creator could be specified by a link and a hash to a Verifiable Credential (or a simple JSON file) that states the creator's name. Storing that name on-chain would create a regulatory problem. Linked Resources allow the information to be shared while ensuring the ability to honor requests for deletion.

example json of a linked resource specifying the creator
```json
"linkedResource" : {
  "id" : "#creator",
  "path": "/creator",
  "rel":"dc:creator",
  "type" : "iid:ResourceDescriptor",
  "proof" : [{
     "type": "hash",
     "stage": "encoded",
     "value" : "bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi"}],
   "resourceFormat" : "application/did+ld+json",
   "compression" : "none",
   "endpoint": "https://earthprogram.directory?listing=did:earth:ixo:nft:1:abc123#owner"
}
```
The VC retrieved from that endpoint contains the personal data describing the author. This personal information is never stored on-chain, rather, it is retrievable from a mutable data store, which retains the ability to honor GDPR deletion requests.

Reply:
I do agree with this. Yours is much better. But I still think there is value in directing the reader to the DID Core spec privacy considerations. Or is it just implicit that we expect the implementor to know about those?